portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,16 +39,8 @@
   @Override
```

Comment:
I think `CommentServlet` or `CommentDataServlet` is more descriptive than DataServlet. Please rename the class and the filename.

Reply:
renamed